### PR TITLE
Use date

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers this is ready for review. The receiver fills date variable automatically

### CODE REVIEW

**Comment by Lunalo** [comment]
This okay. @stevenndungu 

What does this line do?

**Comment by stevenndungu** [comment]
Which line sorry

File: instat/DlgUseDate.vb [hunk 1]
```diff
@@ -53,6 +53,7 @@ Public Class dlgUseDate
         ucrReceiverUseDate.SetMeAsReceiver()
         ucrReceiverUseDate.bUseFilteredData = False
         ucrReceiverUseDate.SetIncludedDataTypes({"Date"})
+        ucrReceiverUseDate.bAutoFill = True
```
Comment (Lunalo) [inline instat/DlgUseDate.vb]: This okay. @stevenndungu 

What does this line do?

Comment (maxwellfundi) [inline instat/DlgUseDate.vb]: @Lunalo this allows for auto filling for data columns when dialog is loaded or reset if it exists in the dataset

Comment (stevenndungu) [inline instat/DlgUseDate.vb]: The line helps in automatically detecting the date data type column specified above in the loaded data frame and if one column is found to be of date data type; it automatically autofills it in the receiver.

Comment (Lunalo) [inline instat/DlgUseDate.vb]: Great. I have used it before in climatic.

Thanks.

**Comment by Lunalo** [comment]
@dannyparsons 

This is ready for your review

**Comment by rdstern** [comment]
That's excellent - the autofilling of the date receiver, when there is just one date column.

**Comment by lilyclements** [comment]
Fixes #2814

**Pull request merged**